jest.setup.js

```diff
@@ -87,3 +87,14 @@ Object.defineProperty(window.HTMLElement.prototype, 'clientHeight', {
         return this._jsdomMockClientHeight || 0
     }
 })
+
+Object.defineProperty(window.HTMLElement.prototype, 'offsetWidth', {
+    get: function () {
+        return this._jsdomMockOffsetWidth || 0
+    }
+})
+Object.defineProperty(window.HTMLElement.prototype, 'offsetHeight', {
+    get: function () {
+        return this._jsdomMockOffsetHeight || 0
+    }
+})
```

Comment:
These are used in the tests to mock buttons having realistic sizes. We were already doing it with `clientHeight`, this is for `offsetHeight`.